src/zzz_od/operation/key_sim_yaml_config.py

```
@@ -0,0 +1,36 @@
+        """
+        self._plugin_dir: Path | None = plugin_dir
+        YamlConfig.__init__(self, config_name, sub_dir=['key_sim'], sample=True,
+                            copy_from_sample=False)
+
+    def _get_yaml_file_paths(self) -> tuple[str | None, str | None, str | None]:
+        # 用户自定义脚本优先，可覆盖插件同名脚本
+        user_dir = os_utils.get_path_under_work_dir('config', 'key_sim')
+        user_yml = os.path.join(user_dir, f'{self.module_name}.yml')
+        if os.path.exists(user_yml):
+            return user_yml, user_yml, None
+        # 其次插件目录内任意位置（目录结构由插件自由组织，不限定子目录名）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)